src/opf/opf.jl

```diff
@@ -1,7 +1,9 @@
 using MathOptSymbolicAD
 using SparseArrays
 
-mutable struct OPFModel{OPF <: PM.AbstractPowerModel}
+abstract type Formulation end
```

Comment:
Julia convention: every abstract type starts with `Abstract`
```suggestion
abstract type AbstractFormulation end
```